client/views/app/sideNav/directMessagesFlex.html

```diff
@@ -14,9 +14,9 @@
 			<div class="input-line">
 				<label for="who">{{_ "With_whom" }}</label>
 				{{#if relabelRoom}}
-					<input id="who" type="text" readonly value="{{selectedUser}}">
```

Comment:
Seems like what is in red is what you want if you're moving from username to user.